hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/StorageBasedLockProvider.java

```diff
@@ -123,7 +125,27 @@ public StorageBasedLockProvider(final LockConfiguration lockConfiguration, final
             lockConfiguration.getConfig(),
             LockProviderHeartbeatManager::new,
             getStorageLockClientClassName(),
-            LOGGER);
+            LOGGER,
+        null);
```

Comment:
Is this constructor still used?

Reply:
in case for some reason the metrics changes get messed up at some later point this will fallback gracefully to the non-metrics constructor